osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -124,9 +125,24 @@ public abstract partial class SongSelect : ScreenWithBeatmapBackground, IKeyBind
         [Resolved]
         internal IOverlayManager? OverlayManager { get; private set; }
 
+        private Bindable<bool> configBackgroundBlur { get; set; } = new BindableBool();
+
         [BackgroundDependencyLoader(true)]
-        private void load(AudioManager audio, OsuColour colours, ManageCollectionsDialog? manageCollectionsDialog, DifficultyRecommender? recommender)
+        private void load(AudioManager audio, OsuColour colours, ManageCollectionsDialog? manageCollectionsDialog, DifficultyRecommender? recommender, OsuConfigManager config)
         {
+            configBackgroundBlur = config.GetBindable<bool>(OsuSetting.SongSelectBackgoundBlur);
+            configBackgroundBlur.BindValueChanged(e =>
+            {
+                if (this.IsCurrentScreen())
+                {
+                    ApplyToBackground(background =>
+                    {
+                        background.IgnoreUserSettings.Value = true;
```

Comment:
This line seems unnecessary.

Reply:
I think this was to prevent a bug, I'm gonna check in a bit if that's really unnecessary

Reply:
If it's preventing a bug, it's very likely that the bug already existed. Also, if it is then this will need test coverage.

Reply:
Nevermind, checked and it must be something that my previous code caused (thought it was something like modifying the other blur setting would override the song select one).